### DISCUSSION TITLE
Does the native image configuration of extensions only apply to Quarkus ？

### DISCUSSION THREAD

**Discussion by yswdqz** [quarkusio/quarkus#34451] [Q&A] [answered]
I'm sorry this question may not be very relevant to the quarkus community.

I recently tried to use a native image to compile my project, but I was confused by how to configure it.

So I have a question that I'd like to ask.
Is the native image configuration of these extensions only apply to Quarkus? For instance, if I were to convert the native image configuration of the gRPC extension into another format, would it be applicable to a gRPC project that doesn't utilize Quarkus ?

Thank you in advance for your time and assistance.

**Comment by maxandersen** [comment, marked as answer]
Not sure what you are asking for? The extensions  and build time optimisations only apply for quarkus apps. 

If you mean if you take the generated config manually apply it you probably will see it work for most cases assuming you use the same grpcapis. 

But we can't guarantee it. 

... But what makes it you can't use quarkus with grpc?

**Reply by yswdqz** [reply]
Thank you for your help !

Because I think it might not be easy to turn a grpc project into a quarkus project?
